.github/workflows/debian.yml

```diff
@@ -7,6 +7,11 @@ on:
       - docker/debian/Dockerfile*
       - test
   workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag suffix to give the Docker images (e.g. v1.2.3)."
```

Comment:
How to verify we're not pushing the same tag again (completely defeating reusable builds)?